src/label-helpers.js

```diff
@@ -34,10 +34,10 @@ function getLabelContent(node) {
 
 // Based on https://github.com/eps1lon/dom-accessibility-api/pull/352
 function getRealLabels(element) {
-  if (element.labels !== undefined) return element.labels
-
   if (!isLabelable(element)) return []
 
+  if (element.labels !== undefined) return element.labels
+
```

Comment:
Is better to do in this way instead of swap the two functions?

```suggestion
  if (element.labels !== undefined) return element.labels ?? []

```

Reply:
I've added a `squash` commit with your suggestion – thanks @marcosvega91.